### PR TITLE
[cdc] Waiver file refining

### PR DESCRIPTION
**Pull request by eunchan**
- Revised waiver syntax. `expression`. to waive the Reconvergence
  Warning

### CODE REVIEW

File: hw/top_earlgrey/cdc/cdc_waivers.gpio.tcl [hunk 1]
```diff
@@ -3,7 +3,12 @@
...
+#  MultiClockDomains=="IO_DIV2_CLK::IO_DIV4_CLK"
 
-set_rule_status -rule { W_RECON_GROUPS } -status { Waived } \
-   -expression {u_gpio.gen_filter} \
-   -comment {filters are reconverged into registers. But it's OK.}
\ No newline at end of file
+set_rule_status -rule { W_RECON_GROUPS } -status { Waived }     \
+   -expression {(ControlSignal=~"*u_gpio.gen_filter*") &&       \
+                (ReconSignal=~"*u_gpio.u_reg.u_intr_state.q*")} \
+   -comment {filters are reconverged into registers. But it's OK.}
```
Comment (tjaychen) [inline hw/top_earlgrey/cdc/cdc_waivers.gpio.tcl:14]: Ah is this the filter output eventually converging into an interrupt going out that something happened?

Comment (eunchan) [reply hw/top_earlgrey/cdc/cdc_waivers.gpio.tcl:14]: Yep. filtering happens in DIV2 and the signal is converged into the status for edge event. PADRING/PINMUX use IO_DIV2 clock that's why I think this reconvergence error occurs.

Comment (tjaychen) [reply hw/top_earlgrey/cdc/cdc_waivers.gpio.tcl:14]: sounds good.

Comment (tjaychen) [reply hw/top_earlgrey/cdc/cdc_waivers.gpio.tcl:14]: Could we just say something along the lines of... the convergence is for status reporting, the events are really all independent?

**Review by tjaychen — APPROVED** [review]

File: hw/top_earlgrey/cdc/cdc_waivers.gpio.tcl (hunk 1 above)
Comment (eunchan) [reply hw/top_earlgrey/cdc/cdc_waivers.gpio.tcl:14]: OK Let me quickly revise it

**Pull request merged**